DESCRIPTION

```diff
@@ -1,20 +1,22 @@
 Package: exploratory
 Type: Package
 Title: R package for Exploratory
-Version: 0.5.5.5
+Version: 0.5.5.7
 Date: 2017-02-03
 Authors@R: c(person("Yosuke", "Yasuda", email = "double.y.919.quick@gmail.com", role = c("aut", "cre")))
 URL: https://github.com/exploratory-io/exploratory_func
 Description: Functions for Exploratory
 License: What license is it under?
 LazyData: TRUE
 Encoding: UTF-8
+Depends:
+    broom,
```

Comment:
Please add comment on criteria on when a package should go to Depends of Imports.